Microsoft.Toolkit.Uwp.UI.Controls/ClassicMenu/ClassicMenu.Events.cs

```diff
@@ -0,0 +1,178 @@
+        }
+
+        private static bool MoveFocusForward(ClassicMenu menu, ClassicMenuItem menuItem)
+        {
+            var currentMenuItemIndex = menu.Items.IndexOf(menuItem);
+            var nextIndex = (currentMenuItemIndex + 1) % menu.Items.Count;
+            var nextItem = menu.Items.ElementAt(nextIndex) as ClassicMenuItem;
+            nextItem?.Focus(FocusState.Keyboard);
+            return true;
+        }
+
+        private static bool MoveFocusBackward(ClassicMenu menu, ClassicMenuItem menuItem)
+        {
+            var currentMenuItemIndex = menu.Items.IndexOf(menuItem);
+            var nextIndex = (currentMenuItemIndex - 1 + menu.Items.Count) % menu.Items.Count;
+            var nextItem = menu.Items.ElementAt(nextIndex) as ClassicMenuItem;
+            nextItem?.Focus(FocusState.Keyboard);
+            return true;
+        }
+
+        private static string MapInputToGestureKey(KeyEventArgs args)
+        {
+            var isCtrlDown = Window.Current.CoreWindow.GetKeyState(VirtualKey.Control).HasFlag(CoreVirtualKeyStates.Down);
+            var isShiftDown = Window.Current.CoreWindow.GetKeyState(VirtualKey.Shift).HasFlag(CoreVirtualKeyStates.Down);
+
+            if (!isCtrlDown && !isShiftDown)
+            {
+                return null;
+            }
+
+            StringBuilder gestureKeyBuilder = new StringBuilder();
+
+            if (isCtrlDown)
+            {
+                gestureKeyBuilder.Append(CtrlValue);
+                gestureKeyBuilder.Append("+");
+            }
+
+            if (isShiftDown)
+            {
+                gestureKeyBuilder.Append(ShiftValue);
+                gestureKeyBuilder.Append("+");
+            }
+
+            if (args.VirtualKey == VirtualKey.None)
+            {
+                gestureKeyBuilder.Remove(gestureKeyBuilder.Length - 1, 1);
+            }
+            else
+            {
+                gestureKeyBuilder.Append(args.VirtualKey);
+            }
+
+            return gestureKeyBuilder.ToString();
+        }
+
+        private void ClassicMenu_Loaded(object sender, RoutedEventArgs e)
+        {
+            var wrapPanel = ItemsPanelRoot as WrapPanel.WrapPanel;
+            if (wrapPanel != null)
+            {
+                wrapPanel.Orientation = Orientation;
+            }
+
+            Dispatcher.AcceleratorKeyActivated -= Dispatcher_AcceleratorKeyActivated;
+            Window.Current.CoreWindow.KeyDown -= CoreWindow_KeyDown;
+            Dispatcher.AcceleratorKeyActivated += Dispatcher_AcceleratorKeyActivated;
+            Window.Current.CoreWindow.KeyDown += CoreWindow_KeyDown;
+        }
+
+        private void ClassicMenu_Unloaded(object sender, RoutedEventArgs e)
+        {
+            Dispatcher.AcceleratorKeyActivated -= Dispatcher_AcceleratorKeyActivated;
+            Window.Current.CoreWindow.KeyDown -= CoreWindow_KeyDown;
+        }
+
+        private void CoreWindow_KeyDown(CoreWindow sender, KeyEventArgs args)
+        {
```

Comment:
How about support for hotkeys on the menu itself? Example: Alt-F to open the File menu, Alt-E to open Edit menu